### ISSUE TITLE
Molecular Grid from Atomic Grids

### ISSUE THREAD

**Issue by PaulWAyers** [theochem/grid#179]
There should be an easy-to-use functionality to get molecular grids from atomic grids, as well as an overall wrapper to take atomic positions and types and construct atomic grids directly, then molecular grids.

**Comment by PaulWAyers** [comment]
A starting point is to add a notebook that shows how to construct a molecular grid from atomic grids.

**Comment by Ali-Tehrani** [comment]
I agree with adding in the notebook on how to construct it using the initializer but I feel like the __init__ method achieves the easy-to-use functionality if the user has an example

https://github.com/theochem/grid/blob/cdf1ec3d4282421968b3db215417f6fd09f486bc/src/grid/molgrid.py#L92-L98 

There is also an example in the code docs of [molecular grid](https://grid.qcdevs.org/pyapi/grid.molgrid.html)

The latter wrapper will depend on the radial grid and consequently the results of the grid assessment.

**Comment by FarnazH** [comment]
- [ ] Address comments 1 and 2 from https://github.com/theochem/grid/pull/199#pullrequestreview-1738493652
- [ ] Double check the 2nd formula in the “Calculate Radial Electron Density Distribution” section
- [ ] Probably we can identify the maximum radial electron density distribution using scipy
